README.md

```diff
@@ -1018,6 +1003,47 @@
   }
   ```
 
+  - don't use ternary operators
+
+  ```javascript
+  // bad
+  var status = response.httpCode === 200 ? 'ok' : 'error';
+
+  // good
+  var status;
+  if (response.httpCode === 200) {
+    status = 'ok';
+  } else {
+    status = 'error';
+  }
+  ```
+  Abusing ternary operators will lead to less readable code, and the same can be accomplished on notation that displays clearer intentions.
+
+  - prefer null coalescing instead of null checking
```

Comment:
This makes things more practical with very little impact, but I understand this would be arguable in some scenarios.